logger/options_test.go

```diff
@@ -69,10 +69,12 @@ func TestOptions_SetDefaults(t *testing.T) {
 		assert.Equal(t, defaultFlushInterval, o.FlushInterval)
 		assert.Equal(t, defaultMaxBufferLen, o.MaxBufferLen)
 		assert.Equal(t, defaultIngestURL, o.IngestURL)
+		assert.Equal(t, time.Now().UnixNano()/1000000, o.Timestamp)
 	})
 
 	t.Run("Retains existing values", func(t *testing.T) {
-		o := Options{FlushInterval: 10 * time.Second, MaxBufferLen: 10, IngestURL: "https://example.org"}
+		var timestamp int64 = 1257894000000
+		o := Options{FlushInterval: 10 * time.Second, MaxBufferLen: 10, IngestURL: "https://example.org", Timestamp: timestamp}
```

Comment:
it can be removed here, it doesn't make sense to be setting a logger-level timestamp